Doc/c-api/exceptions.rst

```diff
@@ -904,7 +904,7 @@ because the :ref:`call protocol <call>` takes care of recursion handling.
 
    Marks a point where a recursive C-level call is about to be performed.
 
-   If :c:macro:`USE_STACKCHECK` is defined, this function checks if the OS
+   If :c:macro:`!USE_STACKCHECK` is defined, this function checks if the OS
    stack overflowed using :c:func:`PyOS_CheckStack`.  In this is the case, it
```

Comment:
"In this case" or "If this is the case"?

Reply:
Got it